src/sjur_klemetsen_ex/ex04/walker.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Sjur Klemetsen'
+__email__ = 'sjkl@nmbu.no'
+
+import random as rd
+
+
+class Walker:
+    def __init__(self, pos, home):
+        self.pos = pos
+        self.home = home
+        self.moves = 0
+
+    def move(self):
+        self.moves += 1
+        i = rd.randint(1, 2)
+        if i == 1:
+            self.pos += 1
+        else:
+            self.pos -= 1
+
+    def is_at_home(self):
+        if self.pos == self.home:
```

Comment:
In case there are not equal the function does not return anything, that can cause problems in other applications
you can write return self.pos == self.home
it will return either true or false